Healthy/Classes/Extensions/UIFont+Style.swift

```diff
@@ -0,0 +1,51 @@
+import UIKit
```

Comment:
Please use the same fonts provided in Figma with exactly the same names. 

https://www.figma.com/file/8ilruINbV0EBVQZP2IQnwB/Food-Recipe-App-(Community)?type=design&node-id=5%3A228&t=EkwGw0Pov8Q6NLVO-1